### ISSUE TITLE
render fails with Fontconfig warning

### ISSUE THREAD

**Issue by ejordon** [faressoft/terminalizer#90]
rendering fails with a Fontconfig warning on Fedora 31 Workstation, using gnome terminal, and terminalizer 0.7.1.

```
$: terminalizer render testrecording.yml 
Error: 
  Error: Fontconfig warning: "/usr/share/fontconfig/conf.avail/05-reset-dirs-sample.conf", line 6: unknown element "reset-dirs"


Hint:
  Use the --help option to get help about the usage
```

**Comment by miraclx** [comment]
The underlying issue is described here
#102

And fixed on PR #97

**Comment by zhiburt** [comment]
I've run into the same issue
```
Error:
  Error: Fontconfig warning: "/usr/share/fontconfig/conf.avail/05-reset-dirs-sample.conf", line 6: unknown element "reset-dirs"
```

```
terminalizer -v
0.7.2
```

**Comment by zoj613** [comment]
Just ran into the same issue on v0.7.2 . Any workaround?

**Comment by imsnif** [comment]
As a workaround I copied the contents of the `render.js` file from the above PR and it worked.

**Comment by jisuchoi01** [comment]
I tried @imsnif 's solution. Copy #97 PR's render.js file and replace on my code.
Same issue on my environment and another warning comes out.

terminalizer : 0.7.2
node : 17.7.2
npm : 8.5.5
Linux archlinux 5.16.15-arch1-1 #1 SMP PREEMPT Thu, 17 Mar 2022 00:30:09 +0000 x86_64 GNU/Linux

```sh
# terminalizer render demo
[render] Fontconfig warning: "/usr/share/fontconfig/conf.avail/05-reset-dirs-sample.conf", line 6: unknown element "reset-dirs"
[render] [4340:0320/040137.569309:FATAL:gpu_data_manager_impl_private.cc(986)] The display compositor is frequently crashing. Goodbye.
Error: 
  Error: Rendering exited with code null

Hint:
  Use the --help option to get help about the usage
```

**Comment by jonas-w** [comment]
Copying the render.js file from PR #97 works for:
terminalizer: 0.7.2
node: v17.8.0
npm: 8.5.5
Linux 5.16.16-zen1-1-zen archlinux
It displays the warnings but renders normally!

**Comment by faressoft** [comment]
The issue appears to have been resolved and released in the new version v0.10.0 by #97. If you believe that there is still something missing or unresolved, please feel free to reopen it.
